src/main/java/teammates/storage/sqlsearch/AccountRequestSearchManager.java

```diff
@@ -50,7 +51,10 @@ void sortResult(List<AccountRequest> result) {
      * Searches for account requests.
      */
     public List<AccountRequest> searchAccountRequests(String queryString) throws SearchServiceException {
-        SolrQuery query = getBasicQuery(queryString);
+        // Escape special characters in the query string
+        String escapedQueryString = ClientUtils.escapeQueryChars(queryString);
+        
+        SolrQuery query = getBasicQuery(escapedQueryString);
```

Comment:
This is an interesting solution. From what I know, the only special characters here are the two `-` characters at the end (the SQL comment), so it is weird that it does anything to escape them. Does this actually get the tests to pass? I cannot tell because the GitHub Actions do not seem to be running. I suspect this is because of the conflicting files. Let's update this branch to the latest commit on the `master` branch and handle the conflicting files.

Regardless, I am also not sure if this can be accepted. Wouldn't this prevent a user from using those special characters when using the search functionality? We may want to see if we can find an alternative way to solve this instead.

Reply:
you are right, it isn't  passing the tests. if I update it all the changes will be lost

Reply:
Since the comment have been escaped,  it shouldn't return 0 responses?

Reply:
if the response is an error the list size should be 0, but what if in our case we don't check it and it accepts the error as a response? That would cause a non-empty-list